common/util/file.py

```diff
@@ -24,6 +24,7 @@ def determine_syntax_files():
 def get_syntax_for_file(filename):
     extension = get_file_extension(filename)
     syntaxes = syntax_file_map.get(filename, None) or syntax_file_map.get(extension, None)
+    # watch out if this one has changed https://github.com/sublimehq/Packages/tree/master/Text
```

Comment:
Oh, I understand now what you intended.   Still, I think it is unnecessary.  After this PR, it isn't very likely that anyone will go looking for `.tmLanguage` in the codebase anyway.  Basically: 6 months from now, I won't know why this comment is here.  And since it won't remove confusion, I say we remove the comment.